juju/model.py

```diff
@@ -1303,7 +1303,7 @@ def _get_series(self, entity_url, entity):
                      (not is_local and 'bundle/' in entity_id))
 
         if is_bundle:
-            handler = BundleHandler(self)
+            handler = BundleHandler(self, trusted=trust, forced=force)
```

Comment:
where does the forced value come from?

Reply:
From the deploy method arguments - https://github.com/juju/python-libjuju/pull/313/files#diff-934c79f928f581e5f1d8903f275de68fR1239